tests/dashboard.spec.ts

```diff
@@ -8,4 +10,18 @@ test.describe('on dashboard screen', () => {
   test('has title', async ({ page }) => {
     await expect(page).toHaveTitle(/Dashboard/);
   });
+  test('Endpoint has been selected randomly', async ({ page }) => {
+    const isAppliedRandomEndpoint = await page.evaluate(() => {
+      return localStorage.getItem('isAppliedRandomEndpoint');
+    });
+    const selectedEndpointObj: string = (await page.evaluate(() => {
+      return localStorage.getItem('selectedEndpoint');
+    })) as string;
+    const selectedEndpoint = JSON.parse(selectedEndpointObj)[0];
+    const isSomeOfAstarEndpoints = providerEndpoints[endpointKey.ASTAR].endpoints.some(
```

Comment:
Maybe to add check that selected endpoint is not light client

Reply:
Ok, I've updated test in [this](https://github.com/AstarNetwork/astar-apps/pull/820/commits/4865bfb7d5d7da7c1975a65be25850ae6aa464b5) commit.